### DISCUSSION TITLE
Add Jinja Template for custom roles

### DISCUSSION THREAD

**Discussion by ccsnw** [aristanetworks/avd#1308] [Q&A] [answered]
Hi guys,

I'd like to add a j2 template and datamodel for custom "roles" that can be assigned to users. 

in EOS you can define:

<role "rolename">
  "sequencenumberA" action"
  "sequencenumberB" action"

and assign this role to a user.

I have created the J2 already and also a datamodel for it. But I'm unsure of what I need to create here in github.

I assume the J2 under "eos_cli_config_gen/templates" , modify eos-intended-config.j2 and also modify the documentation part accordingly.

Am I missing anything? Else I'd like to try implementing this.

Thanks

**Comment by ClausHolbechArista** [comment, marked as answer]
Sorry I missed this back then. The PR is merged by now, so closing discussion.
